### PR TITLE
Feature - Updated docs.Dockerfile to serve docs

### PR DESCRIPTION
**Pull request by MoiyadKayda**
## Checklist:

- [ ] I have updated the CHANGELOG with a summary of my changes
- [ ] I have updated the documentation accordingly

### CODE REVIEW

File: .dockerignore [hunk 1]
```diff
@@ -0,0 +1 @@
+node_modules
```
Comment (krishan711) [inline .dockerignore:1]: remove pls

File: docs.Dockerfile [hunk 2]
```diff
@@ -12,5 +12,9 @@ COPY . .
 RUN npm run build-docs
 
 # Serve with nginx
-FROM ghcr.io/kibalabs/app-serve:latest
+FROM nginx:1.16.0-alpine
```
Comment (krishan711) [inline docs.Dockerfile:15]: this seems wierd - why did this not work with the other image?

Comment (MoiyadKayda) [reply docs.Dockerfile:15]: I am not sure, but I have also not seen the nginx config in the `app-serve` image, I took this config from an youtube video.

**Comment by krishan711** [comment]
Fixed in #103

**Pull request closed without merging**